### DISCUSSION TITLE
Helm Performance Enhancements turn on Asynchronous Options

### DISCUSSION THREAD

**Discussion by meDracula** [DefectDojo/django-DefectDojo#8922] [Q&A] [answered]
Hi, I can't figure out  how to enable with the helm chart the options written in the documentation about [Performance Enhancements](https://documentation.defectdojo.com/usage/performance/).
Yes I understand they are expermentail but still I would like to enable `ASYNC_FINDING_IMPORT` & `ASYNC_OBJECT_DELETE` since I would like see if their is a improved performance difference with large file import and deletion.
Such as files like: [trivy-vuln.json](https://github.com/DefectDojo/sample-scan-files/blob/master/trivy/trivy_v0.2.1_python3.json)

Also I really appreciate your work with Defectdojo :smiley_cat:

**Comment by dsever** [comment, marked as answer]
You can use it by defining env variables DD_ https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/settings/settings.dist.py#L245C9-L245C9

From helm by using https://github.com/DefectDojo/django-DefectDojo/blob/master/helm/defectdojo/values.yaml#L530

**Reply by meDracula** [reply]
Thank you so much :pray: :+1:
